content/F2015/F15-event-teaparty.md

```diff
@@ -0,0 +1,21 @@
+title: Big CSters Board Game and Tea Party Social
+Date: 2015-09-23
```

Comment:
You're going to want to change the date to be after the 24th so that it will show up ahead of the career panel event (which is dated 2015-09-24 in the post) :)